packages/dashboard/src/components/tasks/task-schedule.tsx

```diff
@@ -331,7 +350,8 @@ export const TaskSchedule = () => {
             setEventScope(EventScopes.CURRENT);
             AppEvents.refreshTaskSchedule.next();
           }}
-          submitTasks={submitTasks}
+          onDispatchTask={dispatchTaskCallback}
+          onScheduleTask={editScheduledTaskCallback}
```

Comment:
Can we use the same code for both this and the appbar? The way it is now has quite abit of code smell, `onScheduleTask` has different meaning depending on the context, we need to decouple it, my ideas are either

1. Have 2 different callback explicitly, for create and edit, then both here and appbar can use the same code.
2. Use different components, i.e. have a `EditTask` component, it can use `CreateTaskForm` internally but with some differences in behavior and text.

Reply:
https://github.com/open-rmf/rmf-web/pull/1004/commits/4dece1e03f8d894d0a46602f1b0129e71b55fd68